### ISSUE TITLE
Make progressbar wider

### ISSUE THREAD

**Issue by lightonflux** [nokonoko/Pomf#26]
Hi, my upload speed is very bad. When i upload a big file can't be sure that the file is still uploading without opening my network monitor. pomf's progressbar just doesn't move for several minutes. 

It would be easier if the progress bar was at least double the width it is now.

**Comment by Strat-** [comment]
Or they could implement a percentage meter. I think it'd be way better.

**Comment by nokonoko** [comment]
Didn't you already open a issue about this? Anyways I will not make it wider, I can however ask Nuck if he can switch the bar to % instead since that would look much better.

**Comment by Strat-** [comment]
Yeah, right [here](https://github.com/nokonoko/Pomf/issues/15#issuecomment-53514157), whereas I now change my preference from progress bars to percentages simply because numbers.
